vip_hci/phot/fakecomp.py

```diff
@@ -148,40 +244,41 @@ def create_psf_template(array, size, fwhm=4, verbose=True, collapse='mean'):
         psf = np.median(psf, axis=0)
     else:
         raise TypeError('Collapse mode not recognized.')
-    
+
     psf_normd = psf_norm(psf, size=size, fwhm=fwhm)
-    
-    if verbose:  
-        print("Done scaled PSF template from the average of", n,"frames.")
+
+    if verbose:
+        print "Done scaled PSF template from the average of", n,"frames."
     return psf_normd
 
 
 def psf_norm(array, fwhm=4, size=None, threshold=None, mask_core=None,
              full_output=False, verbose=False):
     """ Scales a PSF, so the 1*FWHM aperture flux equals 1.
-    
+
```

Comment:
@arainot Why there are changes in the psf_norm function at all? It complicates the review. The logic of this function was not supposed to be modified, yet some lines appear to be changed. Please, take care of this in the future, only touch the lines (and characters) that need to be modified.